src/base/dynamixel_bus.cpp

```diff
@@ -24,6 +25,9 @@ class DynamixelBusImpl {
     delete packet_handler_;
   }
 
+  //for bulk read & get position gain
+  std::mutex port_mutex;
```

Comment:
If the Dynamixel SDK handles errors for situations where multiple operations are requested on the same port simultaneously, using a mutex may not be necessary